docs/DEVELOPER_GUIDE.md

```diff
@@ -283,4 +283,42 @@ It`s a red herring. To resolve it, please ensure you have logged into dockerhub
 2019-09-28 01:52:23.345692: E tensorflow_serving/sources/storage_path/file_system_storage_path_source.cc:362] FileSystemStoragePathSource encountered a filesystem access error: Could not find base path /mnt/models for servable flowers-sample
 ```
 
-Please make sure not to deploy the inferenceservice in the `kfserving-system` or other namespaces where namespace has  `control-plane` as a label. The `storage-initializer` init container does not get injected for deployments in those namespaces since they do not go through the mutating webhook.
\ No newline at end of file
+Please make sure not to deploy the inferenceservice in the `kfserving-system` or other namespaces where namespace has  `control-plane` as a label. The `storage-initializer` init container does not get injected for deployments in those namespaces since they do not go through the mutating webhook.
```

Comment:
I wonder if instead of ignoring those namespaces, we should simply only inject if there is a kfserving label on it.

Reply:
This reflects the current behaviour - but I like the approach. @rakelkar can comment from `storage-initializer` perspective

Reply:
regardless - this can be merged

Reply:
I think this is the issue when you create the inference service before the mutating webhook is ready. The webhook configuration should not need to be deleted.

Reply:
@yuzisun which issue you are talking about? Not recommending deleting the webhook config in the above case